panic: runtime error: index out of range

can not use git town sub commands after upgrade to the latest version
```sh
$ git town config

panic: runtime error: index out of range

goroutine 1 [running]:
github.com/Originate/git-town/src/git.(*ConfigMap).initialize(0xc4200442b0)
	/private/tmp/git-town-20180116-61670-tzgeis/git-town-6.0.0/src/github.com/Originate/git-town/src/git/config_map.go:82 +0x3c5
github.com/Originate/git-town/src/git.(*ConfigMap).Get(0xc4200442b0, 0x149bba4, 0x19, 0x0, 0x0)
	/private/tmp/git-town-20180116-61670-tzgeis/git-town-6.0.0/src/github.com/Originate/git-town/src/git/config_map.go:47 +0x2b
github.com/Originate/git-town/src/git.GetConfigurationValue(0x149bba4, 0x19, 0x0, 0x0)
	/private/tmp/git-town-20180116-61670-tzgeis/git-town-6.0.0/src/github.com/Originate/git-town/src/git/config.go:85 +0x41
github.com/Originate/git-town/src/git.GetMainBranch(0x0, 0x0)
	/private/tmp/git-town-20180116-61670-tzgeis/git-town-6.0.0/src/github.com/Originate/git-town/src/git/config.go:95 +0x36
github.com/Originate/git-town/src/git.GetPrintableMainBranch(0x0, 0x0)
	/private/tmp/git-town-20180116-61670-tzgeis/git-town-6.0.0/src/github.com/Originate/git-town/src/git/printable.go:13 +0x22
github.com/Originate/git-town/src/cmd.printConfig()
	/private/tmp/git-town-20180116-61670-tzgeis/git-town-6.0.0/src/github.com/Originate/git-town/src/cmd/config.go:35 +0x45
github.com/Originate/git-town/src/cmd.glob..func6(0x1724cc0, 0x174a9b0, 0x0, 0x0)
	/private/tmp/git-town-20180116-61670-tzgeis/git-town-6.0.0/src/github.com/Originate/git-town/src/cmd/config.go:24 +0x44
github.com/Originate/git-town/vendor/github.com/spf13/cobra.(*Command).execute(0x1724cc0, 0x174a9b0, 0x0, 0x0, 0x1724cc0, 0x174a9b0)
	/private/tmp/git-town-20180116-61670-tzgeis/git-town-6.0.0/src/github.com/Originate/git-town/vendor/github.com/spf13/cobra/command.go:750 +0x2c1
github.com/Originate/git-town/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x1727300, 0x138ea01, 0x14a164f, 0x25)
	/private/tmp/git-town-20180116-61670-tzgeis/git-town-6.0.0/src/github.com/Originate/git-town/vendor/github.com/spf13/cobra/command.go:831 +0x30e
github.com/Originate/git-town/vendor/github.com/spf13/cobra.(*Command).Execute(0x1727300, 0x0, 0x1)
	/private/tmp/git-town-20180116-61670-tzgeis/git-town-6.0.0/src/github.com/Originate/git-town/vendor/github.com/spf13/cobra/command.go:784 +0x2b
github.com/Originate/git-town/src/cmd.Execute()
	/private/tmp/git-town-20180116-61670-tzgeis/git-town-6.0.0/src/github.com/Originate/git-town/src/cmd/root.go:31 +0x3d
main.main()
	/private/tmp/git-town-20180116-61670-tzgeis/git-town-6.0.0/main.go:6 +0x20

```

--- a/src/git/config_map.go
+++ b/src/git/config_map.go
@@ -65,7 +65,7 @@ func (c *ConfigMap) initialize() {
 	if c.initialized {
 		return
 	}
-	cmdArgs := []string{"git", "config", "-l"}
+	cmdArgs := []string{"git", "config", "-lz"}
 	if c.global {
 		cmdArgs = append(cmdArgs, "--global")
 	}
@@ -77,8 +77,11 @@ func (c *ConfigMap) initialize() {
 	if cmd.Output() == "" {
 		return
 	}
-	for _, line := range strings.Split(cmd.Output(), "\n") {
-		parts := strings.SplitN(line, "=", 2)
+	for _, line := range strings.Split(cmd.Output(), "\x00") {
+		if len(line) == 0 {
+			continue
+		}
+		parts := strings.SplitN(line, "\n", 2)
 		key, value := parts[0], parts[1]
 		c.data[key] = value
 	}

